woocommerce/woocommerce
# Remove table of contents from docs

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





* Removes the table of contents from docs as this does not work correctly on the docs site
  * An automatically generated TOC is added to all docs pages on the site



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Verify that the TOC is removed from all markdown files in this commit
2. Verify that there is no longer a TOC in any other markdown files in the docs folder



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
